### ISSUE TITLE
resource time warning

### ISSUE THREAD

**Issue by Riipperino** [apoiat/Esx_Deliveries#2]
When the server is rebooted and you try to pick one of the jobs for the first time,after your clothes changed you get -30 fps and no vehicle will spawn.
After manually /restart esx_deliveries it turns back normal and  vehicle will spawn this time.
Looks like a loadout issue, or skinchanger ?

**Comment by lucasvperini** [comment]
Same here

**Comment by apoiat** [comment]
I noticed this behavior as well but i think it has to do with your ESX player being loaded first. Since you guys are developers you are trying stuff too quick. But for a normal player entering your server that won’t be an issue. Either way I’ll try to form a better preload system in near future

**Comment by DoluTattoo** [comment]
All working fine for me, 32/32 and ESX framework

**Comment by lucasvperini** [comment]
I've set a restart after the resource is loaded on "server.cfg", seems to have fixed the problem.

**Comment by apoiat** [comment]
I still think you tried to start a delivery before your character (including skin) fully loaded into the server.

**Comment by Riipperino** [comment]
> I still think you tried to start a delivery before your character (including skin) fully loaded into the server.

For me this is not the case , it's actually on the end of the ESX list.
Oh and even tried waiting 10 minutes after a server reboot.

**Comment by apoiat** [comment]
happens every time? can you give me a link to your server so i can join and test it? I can't trigger it

**Comment by Lucid123** [comment]
Hello @apoiat 
I have the same problem

**Comment by apoiat** [comment]
Need server address to trigger. Please give me ip and port

**Comment by FrankieM8** [comment]
I also have this issue. i have emailed you my dev ip.

also some F8 errors for ya: https://i.imgur.com/uO0Fup9.png
https://i.imgur.com/q8WKRrP.png


Edit: @apoiat happens when you use the ConVars. Ive just edited and recompiled as a work around.

**Comment by lucasvperini** [comment]
> I also have this issue. i have emailed you my dev ip.
> 
> also some F8 errors for ya: https://i.imgur.com/uO0Fup9.png
> https://i.imgur.com/q8WKRrP.png
> 
> Edit: @apoiat happens when you use the ConVars. Ive just edited and recompiled as a work around.

If i don't use convars, it works?

**Comment by FrankieM8** [comment]
Yup. Don’t use the ConVars. I just edited the source code to how I wanted and recompiled. I tried to fix the convars but I don’t know c# much.

**Comment by apoiat** [comment]
I doubt this has anything to do with convars. Probably a coincidence. I will check on your server

**Comment by Riipperino** [comment]
> Need server address to trigger. Please give me ip and port

Hey sorry for the delayed response , middle in other stuff aswell.
We have a main server and test server , currently the resource is still on test server.
54.36.135.45:30125

Also restarting the resource after server startup will only effect the 1 time for 1 player , after another player comes by and takes the job its going back to resourcetime warning.
